snf-cyclades-app/synnefo/logic/servers.py

```diff
@@ -218,6 +224,7 @@ def create_server(vm, nics, volumes, flavor, image, personality, password):
         'img_format': str(image['format']),
         'img_personality': json.dumps(personality),
         'img_properties': json.dumps(image['metadata']),
+        'auth_keys': public_key,
```

Comment:
I think it is better to make the `auth_keys` optional. This is not the same case as with the personality which defaults to `[]`, which means don't inject any files. Right know `snf-image` will ignore a defined but empty AUTH_KEYS parameter but this is due to the fact that it was decided to append the keys to the file and not rewrite it. It could also mean: Empty the authorized_keys file.